src/System.Runtime/src/System.Runtime.csproj

```diff
@@ -87,16 +95,19 @@
     </Compile>
   </ItemGroup>
   -->
-  <ItemGroup Condition="'$(TargetGroup)' != 'net462' And '$(TargetGroup)' != 'net463'">
-    <ProjectReference Include="..\..\System.Private.Uri\src\System.Private.Uri.csproj">
+  <ItemGroup >
+    <ProjectReference Include="..\..\System.Private.Uri\src\System.Private.Uri.csproj" Condition="'$(TargetGroup)' != 'net462' And '$(TargetGroup)' != 'net463' And '$(TargetGroup)' != 'uap101aot'">
       <OSGroup>Windows_NT</OSGroup>
       <UndefineProperties>%(ProjectReference.UndefineProperties);TargetGroup</UndefineProperties>
     </ProjectReference>
+    <ProjectReference Include="..\..\System.Private.Uri\src\System.Private.Uri.csproj" Condition="'$(TargetGroup)' == 'uap101aot'">
```

Comment:
@SedarG this change looks a little suspicious and likely the reason things aren't working. 

@ericstj should we use a pkgproj reference here?